crates/core/src/db/queries/metadatas.rs

```diff
@@ -264,23 +269,32 @@ pub fn list(
         );
     }
 
-    if let Some(offerers) = offerers {
+    if offerers.is_some() || (with_offers.is_some() && with_offers.unwrap_or(false)) {
```

Comment:
with_offers shouldn't be dependent on offerers. The offerers filter is to find  offers on an nfts for a set of wallets. The with_offers should find any nft that has at least 1 offer for the provided auction houses.

I'd like to see 1 block for offers and 1 for with_offers instead of having them handled in the same block.